user/pages/02.development/docs.md

```diff
@@ -5,151 +5,7 @@ taxonomy:
-
-`[WeatherSkill]`
-
-`api_key = ""`
-
-Put the relevant key in between the quotes and Mycroft Core should begin to use the key immediately.
-
-### API Key services
-
-- [STT API, Google STT](http://www.chromium.org/developers/how-tos/api-keys)
-- [Weather Skill API, OpenWeatherMap](http://openweathermap.org/api)
-- [Wolfram-Alpha Skill](http://products.wolframalpha.com/api/)
-
-These are the keys currently in use in Mycroft Core.
-
-## Configuration
-Mycroft configuration consists of 3 possible config files.
-- `mycroft-core/mycroft/configuration/mycroft.ini`, which lives inside the mycroft codebase/distribution
-- `/etc/mycroft/mycroft.ini`
-- `$HOME/.mycroft/mycroft.ini`
-
-When the configuration loader starts, it looks in those locations in that order, and loads ALL configuration. Keys that exist in multiple config files will be overridden by the last file to contain that config value. This results in a minimal amount of config being written for a specific device/user, without modifying the distribution files.
-
-# Running Mycroft
-## With `start.sh`
-Mycroft provides `start.sh` to run a large number of common tasks. This script uses the virtualenv created by
-`dev_setup.sh`. The usage statement lists all run targets, but to run a Mycroft stack out of a git checkout, the following processes should be started:
-run `./start.sh service`
-run `./start.sh skills`
-run `./start.sh voice`
-
-*Note: The above scripts are blocking, so each will need to be run in a separate terminal session.*
-
-## Without `start.sh`
-
-Activate your virtualenv.
-
-With virtualenv-wrapper:
-```
-workon mycroft
-```
-
-Without virtualenv-wrapper:
-```
-source ~/.virtualenvs/mycroft/bin/activate
-```
-- run `PYTHONPATH=. python client/speech/main.py` # the main speech detection loop, which prints events to stdout and broadcasts them to a message bus
-- run `PYTHONPATH=. python client/messagebus/service/main.py` # the main message bus, implemented via web sockets
-- run `PYTHONPATH=. python client/skills/main.py` # main skills executable, loads all skills under skills dir
-
-*Note: The above scripts are blocking, so each will need to be run in a separate terminal session. Each terminal session will require that the virtualenv be activated. There are very few reasons to use this method.*
-
-# FAQ/Common Errors
-
-#### When running mycroft, I get the error `mycroft.messagebus.client.ws - ERROR - Exception("Uncaught 'error' event.",)`
-
-This means that you are not running the `./start.sh service` process. In order to fully run Mycroft, you must run `./start.sh service`, `./start.sh skills`, and `./start.sh voice`/`./start.sh cli` all at the same time. This can be done using different terminal windows, or by using the included `./mycroft.sh start`, which runs all three process using `screen`.
+# Introduction to Document
+This document is a work in progress and will continually be updated and fixed during the development cycle and as new issues are found and worked on.
```

Comment:
The last part of this flows better as  `... will be continually updated and fixed during the development cycle as we find and work on new issues and features`